bridges/snowbridge/pallets/inbound-queue-v2/src/lib.rs

```diff
@@ -56,6 +55,7 @@ use snowbridge_inbound_queue_primitives::{
 use sp_core::H160;
 use xcm::prelude::{ExecuteXcm, Junction::*, Location, SendXcm, *};
 
+use pallet_bridge_relayers::RewardLedger;
```

Comment:
Pallets should ideally not depend on other pallets, as it introduces problematic tight coupling.

`RewardLedger` is actually defined in the `bp-relayers` primitives crate, and `pallet_bridge_relayers` simply reexports it.

```suggestion
use bp_relayers::RewardLedger;
```

Reply:
Good catch! https://github.com/vgeddes/polkadot-sdk/pull/12/commits/6ae7167648e00cc0f4f78262c3dc715315170b65